tests/test_sh_class.py

```diff
@@ -0,0 +1,128 @@
+"""
+Tests for spherical harmonic class
+"""
+import pytest
+import numpy as np
+import pyfar as pf
+from spharpy.spherical import SphericalHarmonics
+from spharpy.samplings import gaussian, calculate_sampling_weights
+
+def test_sphharm_init():
+    coordinates = pf.Coordinates(1, 0, 0)
+    sph_harm = SphericalHarmonics(n_max=2, coordinates=coordinates)
+    assert sph_harm.n_max == 2
+    assert np.all(sph_harm.coordinates == coordinates)
+
+def test_sphharm_init_invalid_coordinates():
+    with pytest.raises(TypeError):
```

Comment:
We usually match the error message to make sure that the correct error is raised. Can you update that for all tests?
```suggestion
    with pytest.raises(TypeError, match='some string'):
```